tidb-cloud/vector-search-data-types.md

```diff
@@ -118,30 +193,34 @@ There are implicit casts when calling functions receiving vector data types:
 1 row in set (0.01 sec)
 ```
 
-However in arithmetic operations, you need to cast explicitly because the operator accepts any data type so that no implicit cast is applied:
+Sometimes explicit cast is needed when operators or functions accepts multiple data types.
+
+For example, in comparison operators or functions, you may want explicit cast to compare by Vector numeric values instead of comparing by string values:
```

Comment:
```suggestion
Use explicit casts when operators or functions accept multiple data types. For example, in comparisons, use explicit casts to compare vector numeric values instead of string values:
```